addons/functions/functions/fnc_hitEH.sqf

```diff
@@ -0,0 +1,60 @@
+#include "script_component.hpp"
+/*
+ * Author: nomisum
+ * Receiving and calculating Damage
+ *
+ * Arguments:
+ * 0: Trench <OBJECT>
+ * 1: Multiplier <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [TrenchObj, 1] call grad_trenches_functions_fnc_hitEH;
+ *
+ * Public: No
+ */
+
+params ["_trench", "_multiplier"];
+
+if (_trench getVariable [QGVAR(hitHandler), false]) exitWith { 
+    TRACE_1("hitEH","not adding hit handler, already there");
+};
+
+_trench setVariable [QGVAR(hitHandler), true, true];
+
+
+
+_trench addEventHandler ["HitPart", {
```

Comment:
hmm this handler is only added on server. [the docs say](https://community.bistudio.com/wiki/Arma_3:_Event_Handlers#HitPart):
> While you can add "HitPart" handler to a remote unit, the respective addEventHandler command must be executed on the shooter's PC and will only fire on shooter's PC as well.

Sooo… this function should be called on the server AND every client I guess?